CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## v1.5.0
 
 - Add a new option to tracks that support axis: `axisMargin` to add some margin to an axis. See [docs/examples/viewconfs/axis-margin.json](docs/examples/viewconfs/axis-margin.json) for an example
+- Fix #578
```

Comment:
```suggestion
- Fix #578: BarTrack SVG export overplotting error.
```
Just adding a short note about what #578 is so that we don't have to go back to the issues to see.